### PR TITLE
BIOWDL-473: Update naming in tests.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md.
- [x] Documentation was updated (if required).
- [x] Workflow `parameter_meta` was added/updated (if required).

### CODE REVIEW

File: docs/inputs.md [hunk 1]
```diff
@@ -61,6 +61,16 @@
         <i>String &mdash; Default: "quay.io/biocontainers/biowdl-input-converter:0.2.1--py_0"</i><br />
         The docker image used for this task. Changing this may result in errors which the developers may choose not to address.
 </p>
+<p name="SubreadsProcessing.convertDockerImagesFile.memory">
+        <b>SubreadsProcessing.convertDockerImagesFile.memory</b><br />
+        <i>String &mdash; Default: "128M"</i><br />
+        The maximum aount of memroy the job will need.
```
Comment (Redmar-van-den-Berg) [inline docs/inputs.md]: aount -> amount
memroy ->  memory

File: tests/test.pipelines.yml [hunk 2]
```diff
@@ -1,88 +1,229 @@
...
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_24_3p--Sample_24_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_25_3p--Sample_25_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.Sample_26_3p--Sample_26_3p-fastqc
+    - path: test-output/batch_3_august/batch_3_august.fl.json
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.counts
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.report
+    - path: test-output/batch_3_august/batch_3_august.fl.lima.summary
+    - path: test-output/batch_3_august/batch_3_august.fl.stderr.log
 
-    - path: test-output/august/august.flnc.E_1_1_3p--E_1_1_3p.bam
```
Comment (Redmar-van-den-Berg) [inline tests/test.pipelines.yml:24]: Can you add in a few tests with `should_exist: false` to make sure that no polished/`flnc` files are generated when the input is DNA?

**Review by Redmar-van-den-Berg — CHANGES_REQUESTED** [review]
Looks good overall, the renaming of the tests itself and the input files make the code a lot clearer

Found some typo's and requested a change in the tests to make sure DNA reads are not polished

File: docs/inputs.md (hunk 1 above)
Comment (JasperBoom) [reply docs/inputs.md]: Good catch, that is actually a typo in the parameter_meta of the common.wdl file in our tasks repository.

File: tests/test.pipelines.yml (hunk 2 above)
Comment (JasperBoom) [reply tests/test.pipelines.yml:24]: I have also added a `should_exist: false` for the fastqc tasks that handles `fl` files which should not occur in the RNA run.

**Review by Redmar-van-den-Berg — APPROVED** [review]
Thanks!

**Pull request merged**